### PR TITLE
FEAT/Cpc 730 invt ability to add a product to an inventory PT1

### PR DESCRIPTION
**Pull request by nic5694**
JIRA: https://champlainsaintlambert.atlassian.net/browse/CPC-730
## Context:
Added the front end to post a new product
## Changes
- Changes to the structure of the front end 
- Temporairly added the products view with hard coded inventory for testing so my team can add their front end items
This can be bullet point or sentence format.
## Before and After UI (Required for UI-impacting PRs)
Screenshot 2023-09-18 at 10 32 32 PM
Screenshot 2023-09-18 at 10 32 42 PM
## Dev notes (Optional)
- Hardcoded inventory id temporaily for dev will be changed at the end of the sprint on the CPC730 PT2 branch

### CODE REVIEW

File: api-gateway/src/main/resources/static/scripts/fragments/nav.html [hunk 1]
```diff
@@ -20,9 +20,6 @@
             <li class="nav-item">
                 <a class="nav-link" ui-sref="vets">Veterinarians</a>
             </li>
-            <li class="nav-item">
-                <a class="nav-link" ui-sref="rolesDetails">Roles</a>
-            </li>
```
Comment (DylanBrass) [inline api-gateway/src/main/resources/static/scripts/fragments/nav.html:25]: This was done for me thank you !

File: api-gateway/src/main/resources/static/scripts/inventory-product-list/inventory-product-list.controller.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+angular.module('inventoryProductList')
+    .controller('InventoryProductController', ['$http', '$scope', function ($http, $scope) {
+        var self = this;
+        var inventoryId = 1;
+        const productsJSON = [{
+            "id": "6503974c55c4ec4e8ce0fde1",
```
Comment (DylanBrass) [inline api-gateway/src/main/resources/static/scripts/inventory-product-list/inventory-product-list.controller.js:8]: Can't wait to see this with an actual request from the DB

**Review by DylanBrass — APPROVED** [review]
Looks great, just don't forget to remove the hard coded values !

**Review by Eli293 — APPROVED** [review]
Good basic implementation of the frontend, no issues were found on my end.

**Pull request merged**